src/System.Runtime.Extensions/tests/System/Environment.NewLine.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Runtime.Extensions.Tests
+{
+    public class EnvironmentNewLine
+    {
+        [Fact]
+        public void NewLineTest()
+        {
+            //arrange
+            string expectedNewLine = Interop.IsWindows ? "\r\n" : "\n";
```

Comment:
(But it's up to you.  I personally prefer the splitting test aproach whenever there's not a lot of code being duplicated, but both approaches work.)